src/jl_uv.c

```diff
@@ -27,8 +27,23 @@
 #include <io.h>
 #define write _write
 #endif
-#if defined(_COMPILER_MICROSOFT_) && !defined(_Static_assert)
-#define _Static_assert static_assert
+
+#ifndef static_assert
+#  ifdef __cplusplus
+#    if __cplusplus < 201103L
+#      define static_assert
+#    endif
+#  else
+#    define static_assert
+// Remove the following gcc special handling when we officially requires
+// gcc 4.7 (for c++11) and -std=gnu11
+#    ifdef __GNUC__
+#      if __GNUC__ >= 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 6)
```

Comment:
this condition is true on gcc44, even though __GNUC__ = 4 and __GNUC_MINOR__ = 4